### PR TITLE
Auth platform, bifrost, and prod-lv changes

### PR DESCRIPTION
**Pull request by bearfact**

### CODE REVIEW

File: jakelib/deploy.jake [hunk 1]
```diff
@@ -55,7 +62,7 @@ const getAnalyticsCommands = (subName) => {
 
 const apps = {
   // CORE API
-  consoleapi: { cmds: coreRootCommands },
+  'consoleapi': { cmds: coreRootCommands },
```
Comment (tom-con) [inline jakelib/deploy.jake]: do we want a hyphen here to standardize it?

File: jakelib/ecr.jake [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+      cmds.push(`docker tag 052248958630.dkr.ecr.us-west-2.amazonaws.com/${from_stack}-${app_name}:latest 052248958630.dkr.ecr.us-west-2.amazonaws.com/${to_stack}-${app_name}:latest`);
+      cmds.push(`docker push 052248958630.dkr.ecr.us-west-2.amazonaws.com/${to_stack}-${app_name}:latest`)
     });
 
     cmds.push('docker image prune -a -f');
 		
 		jake.exec(cmds, { printStdout: true }, function(){
+      console.log("NOW RESTARTING !!!!")
+      const cluster_name = `${restart_environment}-${to_stack}`;
+      const apps = ['consoleapi', 'web-api', 'worker', 'shipping-api', 'shipping-worker', 'bifrost', 'auth-api', 'console', 'consumer', 'jsreports']
```
Comment (tom-con) [inline jakelib/ecr.jake]: same here

File: jakelib/deploy.jake (hunk 1 above)
Comment (bearfact) [reply jakelib/deploy.jake]: That will take so much work in re-terraforming services.  I thought about it but this is the existing value and didn't want to update it.

Comment (tom-con) [reply jakelib/deploy.jake]: np, we did it in our branch. just wondering if you had too

File: jakelib/ecr.jake [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
-	});
+	// 	jake.exec(cmds, { printStdout: true }, function(){
+  //     apps.forEach(app_name => {
+  //       jake.Task['ecs:restart'].execute(to_cluster, `${to_cluster}-${app_name}`);
+  //       jake.Task['slack:deployment'].execute(to_cluster, app_name);
+  //     });
+
+	// 		complete();
+	// 	});
+	// });
```
Comment (SealedSaint) [inline jakelib/ecr.jake]: Why leave commented? Can we just delete this?

File: README.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ Make sure your .env file is up to date
 1. Navigate to the folder you want to deploy. (Example: `greenchef/services/server-greenchef`)
 2. Check out the branch you want to deploy. (Example: `release/2018-12-04`)
 3. View the `apps` constant in the `deploy.jake` file in `jakelib` to find the correct name of the docker image you want to push to
-(Example: `greenchef/services/server-greenchef` can be deployed to `consoleapi`, `web-api`, `worker`, or `scheduler`)
+(Example: `greenchef/services/server-greenchef` can be deployed to `console-api`, `web-api`, `worker`, or `scheduler`)
```
Comment (SealedSaint) [inline README.md]: Yeah, these changes are a little confusing with some things being `consoleapi` and some being `console-api`. If we can keep it all the same would be nice, but not a big deal.

**Pull request merged**